### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by Rhaegar121**
In this PR, I have completed the following tasks.

1. Added a new species column to the table
2. Inserted 7 animals data
3. Updated the animals table by setting the species column to digimon for all animals that have a name ending in mon.
4. Updated the animals table by setting the species column to pokemon for all animals that don't have species already set.
5. Deleted all records in the animals table, then roll back the transaction.
6. Wrote queries according to the requirements
7. Took all screenshots of the result

<details>
<summary>Screenshots</summary>
<be>

</details>

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -38,4 +38,4 @@ SELECT AVG(weight_kg) FROM animals;
 
 SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
 
-SELECT name, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY name;
+SELECT name, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY name;
```
Comment (Davidosky007) [inline queries.sql:27]: * According to the [project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction,queries.sql%20file.) you are to 
* `verify that change was made`
* roll back the change
* `verify that change was undone`
* Commit the transaction.
* Verify that change was made and persists after commit. 
*Inside a transaction:

> Delete all animals born after Jan 1st, 2022.
> Create a savepoint for the transaction.
> Update all animals' weight to be their weight multiplied by -1.
> Rollback to the savepoint
> Update all animals' weights that are negative to be their weight multiplied by -1.
> Commit transaction
* Below is a screenshot of the highlighted requirements that are missing kindly write queries to answer this conditions 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Rhaegar121 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Proper use of Gitflow 👍 
* Well written readme file 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Rhaegar121) [reply queries.sql:27]: Hi @Davidosky007,  Do I have to insert transaction commands in the queries.sql? Btw I have also verified that changes in the screenshots added in the PR description.

Comment (Rhaegar121) [reply queries.sql:27]: And I have written all the queries in the project requirements except transaction commands. Can you tell me specifically what I missing?

Comment (Rhaegar121) [reply queries.sql:27]: I have updated the queries.sql with transaction commands. Could you please kindly check it and remind me if I miss something? @Davidosky007

File: queries.sql [hunk 2]
```diff
@@ -1,41 +1,26 @@
...
 SELECT COUNT(*) FROM animals;
-
 SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
-
 SELECT AVG(weight_kg) FROM animals;
-
 SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
-
-SELECT name, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY name;
+SELECT name, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY name;
```
Comment (1GORDON) [inline queries.sql:27]: Great job @Rhaegar121 🤝

- [x] [Optional] You may also add this query to your `queries.sql` file
query

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Rhaegar121 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (Rhaegar121) [reply queries.sql:27]: Thanks I totally forgot that to insert!

**Pull request merged**